README.md

```diff
@@ -365,39 +188,18 @@ Installation Instructions
       - core/precheck
       - core/node
       - core/cluster
-      - gui/precheck
-      - gui/node
-      - gui/cluster
-      - zimon/precheck
-      - zimon/node
-      - zimon/cluster
-      - callhome/precheck
-      - callhome/node
-      - callhome/cluster
-      - callhome/postcheck
+      - core/postcheck
   ```   
 
-  The following installation methods are available:
+  Again, this is just a minimal example. There are different installation methods available, each offering a specific set of options:
 
-  - Installation from (existing) YUM repository (`scale_install_repository_url`)
-  - Installation from remote installation package (`scale_install_remotepkg_path`)
-  - Installation from local installation package (`scale_install_localpkg_path`)
-  - Installation from single directory package path (`scale_install_directory_pkg_path`)
+  - Installation from (existing) YUM repository (see [samples/playbook_repository.yml](samples/playbook_repository.yml))
+  - Installation from remote installation package (see [samples/playbook_remotepkg.yml](samples/playbook_remotepkg.yml))
+  - Installation from local installation package (see [samples/playbook_localpkg.yml](samples/playbook_localpkg.yml))
+  - Installation from single directory package path (see [samples/playbook_directory.yml](samples/playbook_directory.yml))
```

Comment:
This file doesn't seem to exist...  am i wrong?

Reply:
You mean `samples/playbook_directory.yml`? Yes, that one doesn't exist - yet. Did I mention that it's still "work in progress" ;-) The others should exist, though...

Reply:
I've added the missing sample, hope that that was the last dead link... Thanks for spotting it!